### DISCUSSION TITLE
Please check my assumptions

### DISCUSSION THREAD

**Discussion by twack** [aqualinkd/AqualinkD#275] [Q&A] [answered]
So I will embark on the hardware setup tomorrow. I have the software installed. I would appreciate if someone wouldn't mind reviewing my assumptions on the hardware assy side. I have the RS6 Panel that will be operated on. I'm also curious on how you all have placed the PI. Since my RS6 Box is metal, I assume its going to act as a faraday cage.

Parts List:
* Pi Zero 2 W
* Ethernet/USB HUB HAT Expansion Board for Raspberry Pi 4 B/3 B+/3 B/2 B/Zero/Zero W /2 W/Zero WH/2WH
* FTDI USB-RS485-WE-1800-BT Cable, USB to RS485 Serial
* PW-D Control Buck Converter 6-24V to 5V
* Micro USB Male 2 Pin Bare Wire,1M/3.3FT Micro USB to Open End Pigtail

Assumed hardware assy steps:
1. Solder hot and ground from Micro USB to Buck Inverter output
2. Solder hot and ground input of buck invert together with hot and ground from USB-RS485 Cable.
3. Install the hots and grounds, along with Data +/- USB-RS485 wires into the control panel bus
4. Plug USB-A style into USB Hat on PI
5. Plug Micro-USB into power port of Pi zero itself

Your responses are greatly appreciated.

**Comment by sfeakes** [comment]
That all sounds correct, You don't need to put the Pi in the RS6 box. Most people have a some form of keypad inside the house that has the RS485 wires running to it, so you can piggyback off those.  I did use a Pi in the RS6 box (for a different project) and while the pi was fine, the HAT's would usually crap out after 6 months or so due to heat (I live in TX and inside the box would regularly get to ~120deg). HAT's usually don't have the same build quality as the Pi itself. 
What are you using a HAT for? (a wired ethernet connection)?  it sounds like that's the only reason you need it.

**Reply by twack** [reply]
How else do I get the usb-A into the pi?

I do not have a keypad inside. I want to use wifi to the pi.

**Reply by sfeakes** [reply, marked as answer]
They make adapter cables.

like this. ( I’ve used this specific one, just first that came up on search)
https://www.amazon.com/UGREEN-Adapter-Samsung-Controller-Android/dp/B00N9S9Z0G/

**Reply by twack** [reply]
That's the ticket! I do not know why I was stuck in having the power and signal on different usb's. I'll drop the hat and mount the Pi outside the RS6 enclosure in one of these: https://www.amazon.com/Joinfworld-Outdoor-Electrical-Waterproof-Enclosure/dp/B0CHHJ49QN.

I'll post the final install and actual parts used in a couple of days.

**Reply by sfeakes** [reply]
That box looks like it’ll do the job well.  Their is also a ton of space inside the RS cabinet, I had my Pi in their at one point and it was an easy fit. I also didn’t have any wifi issues being in the cabinet. (I forgot to answer that question earlier).
